File: loops/loops.js
console.log("Loops")

//    start     stop  step
for (let i = 0; i < 10; i+=2){
    console.log(i)
}

const fruits = ["apple", "pear", "tomato"]

for(let i = 0; i < fruits.length; i++){
    console.log(i, fruits[i]);
}

let counter = 10;
while (counter > 0){
    console.log(counter);
    counter--;
}

do{
    console.log("in do while")
}while(false);

while (false){
    console.log("in while")
}

File: loops/loops.py
# for i in range(10):
#     print(i)

for i in range(2, 10, 2):
    print(i)

fruits = ["apple", "pear", "tomato"]

print("Fruits")
for i in range(len(fruits)):
    print(i, fruits[i])

for index, value in enumerate(fruits):
    print(index, value)

counter = 10
while counter > 0:
    print(counter)
    counter -= 1